lib/wallet/integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -198,102 +200,66 @@ spec :: forall n.
-        \ctx -> runResourceT $ do
-
-        wSrc <- fixtureWallet ctx
-        wDest <- emptyWallet ctx
-
-        let amt = minUTxOValue (_mainEra ctx) - 1
-        addrs <- listAddresses @n ctx wDest
-        let destination = (addrs !! 1) ^. #id
-        let payload = Json [json|{
-                "payments": [{
-                    "address": #{destination},
-                    "amount": {
-                        "quantity": #{amt},
-                        "unit": "lovelace"
-                    }
-                }],
-                "passphrase": #{fixturePassphrase}
-            }|]
+    it "TRANS_MIN_UTXO_01: \
 
-        let ep = Link.createTransactionOld @'Shelley
-        r <- request @(ApiTransaction n) ctx (ep wSrc) Default payload
-        expectResponseCode HTTP.status403 r
-        expectErrorMessage errMsg403MinUTxOValue r
+        \Specifying a non-zero quantity of lovelace that is below the minimum \
+        \required by the ledger results in an error, but specifying a quantity \
+        \that is equal to the required minimum results in success." $
 
-    it "TRANS_MIN_UTXO_BABBAGE_VALIDATION - I can spend exactly minUTxOValue" $
         \ctx -> runResourceT $ do
-        -- Coin selection may, when itself /calculating and assigning/
-        -- minUTxOValue to an output, assign a value in excess of the actual
-        -- requirement. Currently (Aug 2022) the excess is in practice always
-        -- 0.017240 ada (c.f. calculation below).
-        --
-        -- This test checks that the excess is not required when /validating/
-        -- user-specified output ada quantities. We don't want the wallet's
-        -- validation to be stricter than the ledger's.
-        wSrc <- fixtureWallet ctx
-        wDest <- emptyWallet ctx
-
-        when (_mainEra ctx < ApiBabbage) $
-            liftIO $ pendingWith "only for the babbage era or later"
-
-        addrs <- listAddresses @n ctx wDest
-        let destination = (addrs !! 1) ^. #id
 
-        let ep = Link.createTransactionOld @'Shelley
-
-        minAda <- counterexample
-            "1. calculate the exact minUTxOValue (for address length)\
-            \ and manually compensate for the overestimation" $ do
-            r <- request @(ApiTransaction n) ctx (ep wSrc) Default $
-                Json [json|{
-                    "payments": [{
-                        "address": #{destination},
-                        "amount": {
-                            "quantity": 0,
-                            "unit": "lovelace"
-                        }
-                    }],
-                    "passphrase": #{fixturePassphrase}
-                }|]
+        sourceWallet <- fixtureWallet ctx
+        targetWallet <- emptyWallet ctx
+        targetAddresses <- listAddresses @n ctx targetWallet
+        let targetAddress = (targetAddresses !! 1) ^. #id
+        let endpoint = Link.createTransactionOld @'Shelley sourceWallet
```

Comment:
We test [postTransaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransaction), but the change concerns more endpoints actually, like:
 - [postTransactionFee](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransactionFee)
 - [postTransaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransaction)
 - [postByronTransactionFee](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postByronTransactionFee)
 - [postByronTransaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postByronTransaction)
 - [constructTransaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/constructTransaction)
 - [constructSharedTransaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/constructSharedTransaction)

Do you think it makes sense to expand this integration test or add versions of it to sub-suites for Byron, Shelley, Shared wallets?

Reply:
> We test [postTransaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransaction), but the change concerns more endpoints actually, like:
> 
>     * [postTransactionFee](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransactionFee)
> 
>     * [postTransaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransaction)
> 
>     * [postByronTransactionFee](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postByronTransactionFee)
> 
>     * [postByronTransaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postByronTransaction)
> 
>     * [constructTransaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/constructTransaction)
> 
>     * [constructSharedTransaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/constructSharedTransaction)
> 
> 
> Do you think it makes sense to expand this integration test or add versions of it to sub-suites for Byron, Shelley, Shared wallets for aforementioned eps?

We could do, though would you be open to us tackling these as separate tasks? There are quite a few integration tests with old/out-of-date methods of computing minimum UTxO values, and I'd hope to tackle those too! I hope to create a JIRA technical debt ticket for this first thing next week. Hopefully we could assign it by next sprint at the latest. Would that work for you?